x-pack/plugins/apm/public/components/app/Main/UpdateBreadcrumbs.tsx

```diff
@@ -8,11 +8,8 @@ import { Location } from 'history';
 import { last, pick } from 'lodash';
 import React from 'react';
 import chrome from 'ui/chrome';
-import {
-  fromQuery,
-  PERSISTENT_APM_PARAMS,
-  toQuery
-} from '../../shared/Links/url_helpers';
+import { fromQuery, toQuery } from '../../shared/Links/url_helpers';
+import { PERSISTENT_APM_PARAMS } from '../../shared/Links/APMLink';
 import { Breadcrumb, ProvideBreadcrumbs } from './ProvideBreadcrumbs';
 import { routes } from './routeConfig';
```

Comment:
I noticed that the `persistentParams` doesn't take the defaults into account:

```js
const persistentParams = pick(query, PERSISTENT_APM_PARAMS);
```

Is that intentional? 
(maybe it is, just wanted to make sure, since we do take the defaults into account in  `getSearchWithCurrentTimeRange`)

Reply:
Ah, not sure. But this is basically just an APM link and should work exactly the same -- I may just export the APM href creation from APMLink instead of doing it differently here.